.env.example

```diff
@@ -0,0 +1,2 @@
+  GH_TOKEN=
+  REDIS_URL=
```

Comment:
Should this maybe list the default redis url, so you don't think you _need_ to set it?

`redis://localhost:6379`

Reply:
✅